model/scan.py

```diff
@@ -155,6 +156,11 @@ def make_scan(
     def single(filename, format, exposure_times, osc_start, osc_width, epoch):
         """Construct an scan instance for a single image."""
 
+        if format is not None:
+            warnings.warn(
+                "using format in scan is now deprecated", warnings.DeprecationWarning
+            )
```

Comment:
I'd prefer
```suggestion
            warnings.warn(
                "using format in scan is now deprecated", warnings.DeprecationWarning, stacklevel=2
            )
```
as this flags the warning at the point where `single()` is called, rather than the point where `warnings.warn` itself is called